src/webargs/core.py

```diff
@@ -503,7 +515,7 @@ def load_files(self, req, schema):
         return missing
 
     def handle_error(
-        self, error, req, schema, error_status_code=None, error_headers=None
+        self, error, req, *, schema, error_status_code=None, error_headers=None
```

Comment:
Should `schema` be a required keyword-only arg?
I would have left all required arguments as positionals and made only the optional arguments (`error_status_code`, `error_headers`) keyword-only.
I also see some variation in the signatures for this in the various parsers, which I assume is accidental.

This is not critical, since `handle_error` is only really called internally and exposed as a hook for users to write something which gets called by webargs. But the two signatures which occur to me are
```python
# everything keyword-only
def handle_error(
    self, *, error, req, schema, error_status_code=None, error_headers=None
):
# optional args keyword-only
def handle_error(
    self, error, req, schema, *, error_status_code=None, error_headers=None
):
```